### ISSUE TITLE
[Enhancement Request] run parallel features in cucumber

### ISSUE THREAD

**Issue by hemanth-dunzo** [cucumber/docs#350]
Currently thread count value that the user provides runs the scenarios in parallel by default.
It would be better as an enhancement to run features in parallel.

**Comment by luke-hill** [comment]
Can you link to where this is specified. I was under the impression that we didn't provide features to run in parallel by default (But I could be mistaken)

**Comment by hemanth-dunzo** [comment]
@luke-hill 
I guess I wasn't clear enough. Let me tell you my concern. 
If we provide --threads 2 then this runs the scenarios in parallel.

As an enhancement request, it would be better to give the user an option with run level = scenarios (which runs scenarios in parallel) or run level = features (which runs the features in parallel)

Example of the enhancement request:
-Dcucumber.options="--threads=3 runLevel=scenarios" will run the scenarios in parallel
-Dcucumber.options="--threads=3 runLevel=features" will run the features in parallel
-Dcucumber.options="--threads=3 runLevel=tags" will run the tags in parallel

**Comment by luke-hill** [comment]
I'm still a bit confused, is this a documentation issue that you feel could be better worded or an actual physical change you want to make to the cucumber binary (I'm assuming you're using Java)

**Comment by hemanth-dunzo** [comment]
An actual change if we don't have an option
Do we have an option to run features in parallel or 
tags in parallel?
If so then this is a documentation issue.

**Comment by luke-hill** [comment]
Right, can you go over to the cucumber-jvm repo.

I see you've already opened an issue there, so I'll close this as a duplicate of https://github.com/cucumber/cucumber-jvm/issues/1651

**Comment by hemanth-dunzo** [comment]
@luke-hill correct. I wanted to know if the feature already exists. If so, I will close the bug in the repo.
